pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -121,9 +117,11 @@ def __init_monitor_mode(self):
     def _validate_condition_metric(self, logs):
         monitor_val = logs.get(self.monitor)
 
-        error_msg = (f'Early stopping conditioned on metric `{self.monitor}`'
-                     f' which is not available. Pass in or modify your `EarlyStopping` callback to use any of the'
-                     f' following: `{"`, `".join(list(logs.keys()))}`')
+        error_msg = (
+            f'Early stopping conditioned on metric `{self.monitor}`'
```

Comment:
There are single quotes being used at some places?

Reply:
yeah @Borda I also thought the formatter would standardize that?

Reply:
seems that yapf doe not force string, we may ask it as a feature :]